pkg/boot/multiboot/module.go

```diff
@@ -145,3 +157,21 @@ func (m modules) marshal() ([]byte, error) {
 	err := binary.Write(&buf, ubinary.NativeEndian, m)
 	return buf.Bytes(), err
 }
+
+// elems returns the elements expected by the mutiboot info header.
```

Comment:
It might just be me but I was initially a little confused about the difference between this comment and the one for memoryMaps elems(), could you explain it a little more?